stl/inc/xhash

```diff
@@ -277,9 +283,10 @@ struct _Hash_vec {
         if (_Oldsize < _Cells) {
             const auto _Newvec = _Alvec.allocate(_Cells); // throws
             // nothrow hereafter
-            if (_Oldsize != 0) {
+            const auto _Oldcapacity = capacity();
+            if (_Oldcapacity != 0) {
                 _Destroy_range(_Mypair._Myval2._Myfirst, _Mypair._Myval2._Mylast);
-                _Alvec.deallocate(_Mypair._Myval2._Myfirst, _Oldsize);
+                _Alvec.deallocate(_Mypair._Myval2._Myfirst, _Old_capacity);

```

Comment:
```
C:\a\1\s\stl\inc\xhash(289,61): error C3861: '_Old_capacity': identifier not found
                _Alvec.deallocate(_Mypair._Myval2._Myfirst, _Old_capacity);
                                                            ^
C:\a\1\s\stl\inc\xhash(319): note: see reference to class template instantiation 'std::_Hash_vec<_Aliter>' being compiled
```

Reply:
"Should I use `_Old_capacity` to be consistent with modern naming conventions or `_Oldcapacity` to be locally consistent? Why not both?" 🤦